tests/postgres_tests/test_search.py

```diff
@@ -286,3 +286,20 @@ def test_ranking_chaining(self):
             rank=SearchRank(SearchVector('dialogue'), SearchQuery('brave sir robin')),
         ).filter(rank__gt=0.3)
         self.assertSequenceEqual(searched, [self.verse0])
+
+
+@skipUnlessDBFeature('has_phraseto_tsquery')
+@modify_settings(INSTALLED_APPS={'append': 'django.contrib.postgres'})
+class TestPhraseSearchQuery(GrailTestData, PostgreSQLTestCase):
+    def test_simple_phrase_search(self):
+        searched = Line.objects.filter(
+            dialogue__search=PhraseSearchQuery('body burned'))
+        self.assertSequenceEqual(searched, [self.verse1])
+
+
+@modify_settings(INSTALLED_APPS={'append': 'django.contrib.postgres'})
+class TestRawSearchQuery(GrailTestData, PostgreSQLTestCase):
+    def test_simple_raw_search(self):
+        searched = Line.objects.filter(
+            dialogue__search=RawSearchQuery('shall & use'))
+        self.assertSequenceEqual(searched, [self.bedemir0])
```

Comment:
You might want to write some tests to prove that the new query search subclasses combine correctly with SearchQuery.